### ISSUE TITLE
edit-event: switch to using forms

### ISSUE THREAD

**Issue by nickolashe** [opencurrents/opencurrents#349]
currently all input elements hardcoded in html

**Comment by dannypernik** [comment]
@nickolashe does this mean we are not yet handling errors on edit-event?

**Comment by nickolashe** [comment]
exactly, we are parsing the raw request data, violating guideline 2 from best coding practices:
https://github.com/opencurrents/opencurrents/blob/develop/openCurrents/views.py#L1303
